oops/oops.go

```diff
@@ -6,10 +6,26 @@ import (
 	"bytes"
 	"fmt"
 	"io"
+	"os"
 	"runtime"
 	"strings"
 )
 
+var (
+	// filePrefixesToSkip is a set of prefixes of files which, when encountered when serializing
+	// an oops error, will skip appending the current frame and all subsequent frames in that stack.
+	// It is useful for truncating long call stacks to make logged oops errors more concise.
+	filePrefixesToSkip = make(map[string]struct{})
+)
+
+func init() {
+	envPrefixesToSkip := os.Getenv("OOPS_FILE_PREFIXES_TO_SKIP")
```

Comment:
What would you think about naming the env var something like `OOPS_SHORT_CIRCUIT_FILE_PREFIXES` or `OOPS_FILE_PREFIXES_AT_WHICH_TO_TRUNCATE` (lol)? I don't think "to skip" intuitively conveys that all subsequent frames are also skipped.